### DISCUSSION TITLE
Edit Sidepanel from Tableblock

### DISCUSSION THREAD

**Discussion by oich** [Budibase/budibase#9212] [Help] [answered]
Hi

Can i edit the Sidepanel of a Tableblock which i reach wnn i click on the "Create Row" button or on a Row ?
I talk about the Sidepanels which i can activate in the settings of the Tableblock ("Show Detail Sidepanel" and "Show new Row Sidepanel")

**Comment by oich** [comment, marked as answer]
I found it 
I Have to "Eject" the Table block in the settings of the Table block

**Comment by diegopaulin** [comment]
Same problem here. Thanks!
